### DISCUSSION TITLE
Not able to deploy through github pages.

### DISCUSSION THREAD

**Discussion by Abhizzek** [community/community#117502] [Programming Help]
### Body

Hello, I am new to github and as I was trying to deploy my potfolio, it is showing me an error. I tried a lot to de bug the error and to resolve it but to no avail. I hope the  community would contribute to resolve my problem Thank you.

![Screenshot 2024-04-07 105800](https://github.com/community/community/assets/148735249/56e2adfe-22f3-4170-92ff-bdee242c4755)
![Screenshot 2024-04-07 105817](https://github.com/community/community/assets/148735249/3b8e1176-294d-4bc6-8f15-81104e3faaae)
![Screenshot 2024-04-07 105825](https://github.com/community/community/assets/148735249/d5e62d41-1e84-4915-a198-16f1d24caa4e)

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by eranga-mohotty** [comment]
Hello @Abhizzek ,
Are you using jekyll? **If not** comment out every thing in config.yml file. Also make sure you have an index.html and if you go to your repo settings>Pages 
under source select Deploy from Branch  (keep Branch settings as main and root respectively if you need the source to be default branch)
Note: I'm assuming you are trying to host a static website with a index.html starter page and not using jekyll

![image](https://github.com/community/community/assets/32839699/89d01588-45a0-4ecf-8457-4153054c841b)
refer above image for settings

**Comment by Abhizzek** [comment]
Firstly, Thank you @eranga-mohotty   for your prompt response, actually I have watched the youtube video of https://www.youtube.com/watch?v=ocdwh0KYeUs and followed his steps. Unfortunately I am unaware about the location of config.yml file and also regarding jekyll, but I possibly might be facing issue due to jekyll. 
![Screenshot 2024-04-07 112110](https://github.com/community/community/assets/148735249/826bc021-23fd-4786-8852-049998f10388)
![Screenshot 2024-04-07 112005](https://github.com/community/community/assets/148735249/c8a4891e-b40f-4836-9013-c8b13a64efb0)

**Comment by jfullstackdev** [comment]
upon checking, it's successful but you did not get the correct path, this is the correct URL because you don't have the correct repo name, so it does not host in the root :

https://abhizzek.github.io/AbhishekDataAnalyst.github.io/

![image](https://github.com/community/community/assets/47092464/43165995-9599-4480-a18e-0f7d6f14673b)
